server/play-dgraph-io.nginx.conf

```diff
@@ -1,5 +1,5 @@
 proxy_cache_path /var/nginx levels=1:2 keys_zone=dgraphcache:20m max_size=1g inactive=60m use_temp_path=off;
-limit_req_zone $binary_remote_addr zone=play:10m rate=1000r/s;
+limit_req_zone $binary_remote_addr zone=play:10m rate=800r/s;
```

Comment:
any reason to change from `1000` to `800`? has there been any issue related to this? an additional `200` doesn't seem to be an issue imo.

Reply:
I'm not sure, but that change was made manually by Daniel.